### ISSUE TITLE
Add contact form

### ISSUE THREAD

**Issue by carolinemao** [acf-quizbowl/acf-quizbowl.github.io#33]
- [ ] List all ACF officers/editors' emails
- [ ] (+ misconduct form?)
- [ ] Require email
- [ ] Require first and last name

**Comment by hftf** [comment]
We decided to get rid of the contact form last year as it was broken and redundant (see 8fc27f4c 352d64dc), but maybe that was just a problem with the form service

**Comment by carolinemao** [comment]
yeah I know haha I was the one who requested for it to be put there—I still think it's useful to help solve people's allergy to email (and I will try to add a working one this time!!)
